index.html

```diff
@@ -4669,105 +4714,130 @@ <h3>Data First Approaches</h3>
     <h2>Internationalization Considerations</h2>
 
     <p>
-There are a number of internationalization considerations implementers should be
-aware of when publishing data described in this specification. As with any web
-standards or protocols implementation, ignoring internationalization makes it
-difficult for data to be produced and consumed across a disparate set of
-languages and societies, which would limit the applicability of the
-specification and significantly diminish its value as a standard.
+There are a number of internationalization considerations implementers
+are advised to be aware of when publishing data described in this specification.
+As with any web standards or protocols implementation, ignoring
+internationalization makes it difficult for data to be produced and consumed
+across a disparate set of languages and societies, which would limit the
+applicability of the specification and significantly diminish its value as
+a standard.
+    </p>
+
+    <p>
+Implementers are strongly advised to read the Strings on the Web: Language and
+Direction Metadata document [[STRING-META]] published by the W3C
```

Comment:
```
the Strings on the Web: Language and
Direction Metadata document
```
->
```
the <em>Strings on the Web: Language and
Direction Metadata</em> document
```

Reply:
I just published an update of the Strings on the Web doc in /TR at https://www.w3.org/TR/string-meta/.  You may want to point to that location, instead of the ED.  I'll try to update regularly using echidna when we add new text to the ED.

Reply:
@TallTed done in f0b4f5063bad9633a8d0c5407fac3a2408ea4121.

#r12a done in f0b4f5063bad9633a8d0c5407fac3a2408ea4121.